index.js

```diff
@@ -156,27 +157,64 @@ module.exports = {
     "space-unary-ops": "error",
 
     // enforce consistent spacing after the // or /* in a comment, and before the */
-    "spaced-comment": ["error", "always", { "line": { "markers": ["/"]}, "block": { "balanced": true}}],
+    "spaced-comment": ["error", "always", { "line": { "markers": ["/"] }, "block": { "balanced": true } }],
 
     // require the jsdocs to be valid
     "valid-jsdoc": ["error", {
       "requireParamDescription": false,
       "requireReturnDescription": false,
 
       // only require a return if there is one in the function
-      "requireReturn": false,
+      "requireReturn": false
+    }],
 
+    // Rules from jsdoc plugin
+    "jsdoc/check-alignment": 1, // Recommended
```

Comment:
We prefer using `off/warn/error` strings instead of `0/1/2`, because it's more intuitive to someone unfamiliar with ESLint configuration.

We also generally prefer using `error` over `warn` because warnings are easily ignored, or missed due to the volume of output in the build. In this case, `warn` could be beneficial to help ease the transition for other projects if they want to upgrade but don't have the immediate resources to fix the JSDoc changes. @wallw-bits do you have a preference on that? Personally I would suggest using `error` given most of the issues are resolved with `--fix`.

Reply:
I went through the upgrade process on a few reasonably large projects using OpenSphere and 99.9% of the issues were resolved with `--fix`. I think I had 3-4 that weren't in each project. Let's go ahead and set all of the enabled rules to `error`.

Reply:
Fixed.